pandas/core/generic.py

```diff
@@ -5909,6 +5909,100 @@ def infer_objects(self: FrameOrSeries) -> FrameOrSeries:
             )
         ).__finalize__(self)
 
+    def as_nullable_dtypes(
+        self: FrameOrSeries, convert_integer: bool_t = True
+    ) -> FrameOrSeries:
+        """
+        Convert columns of DataFrame or a Series to dtypes supporting ``pd.NA``.
+
+        Parameters
+        ----------
+        convert_integer : bool, default True
```

Comment:
why only convert_integer? should there be a convert_bool as well?

Reply:
> why only convert_integer? should there be a convert_bool as well?

Probably yes